docs/public/specifications/tests/DNSSEC-TP/dnssec10.md

```diff
@@ -81,16 +81,16 @@ DS10_NO_DNSSEC_SUPPORT             | NOTICE  | ns_list    | The zone is not DNSS
 DS10_NSEC3PARAM_QUERY_RESPONSE_ERR | ERROR   | ns_ip_list | No response or error in response on query for NSEC3PARAM. Fetched from the nameservers with IP addresses "{ns_ip_list}".
 DS10_NSEC3PARAM_GIVES_ERR_ANSWER   | ERROR   | ns_ip_list | Unexpected DNS record in the answer section on an NSEC3PARAM query. Fetched from nameservers with IP addresses "{ns_ip_list}".
 DS10_NSEC3_ERR_TYPE_LIST           | ERROR   | ns_ip_list | NSEC3 record for the zone apex with incorrect type list. Fetched from nameservers with IP addresses "{ns_ip_list}".
-DS10_NSEC3_MISMATCHES_APEX         | ERROR   | ns_ip_list | The NSEC3 record returned does not match the zone name, which it should. Fetched from nameservers with IP addresses "{ns_ip_list}".
+DS10_NSEC3_MISMATCHES_APEX         | ERROR   | ns_ip_list | The returned NSEC3 record unexpectedly does not match the zone name. Fetched from nameservers with IP addresses "{ns_ip_list}".
 DS10_NSEC3_MISSING_SIGNATURE       | ERROR   | ns_ip_list | Missing RRSIG (signature) for the NSEC3 record or records. Fetched from the nameservers with IP addresses "{ns_ip_list}".
 DS10_NSEC3_NODATA_MISSING_SOA      | ERROR   | ns_ip_list | Missing SOA record in NODATA response with NSEC3. Fetched from nameservers with IP addresses "{ns_ip_list}".
-DS10_NSEC3_NODATA_WRONG_SOA        | ERROR   | ns_ip_list | Wrong owner name on SOA record in NODATA response with NSEC3. Fetched from nameservers with IP addresses "{ns_ip_list}".
+DS10_NSEC3_NODATA_WRONG_SOA        | ERROR   | ns_ip_list, domain | Wrong owner name ({"domain"}) on SOA record in NODATA response with NSEC3. Fetched from nameservers with IP addresses "{ns_ip_list}".
 DS10_NSEC3_RRSIG_VERIFY_ERROR      | ERROR   |ns_ip_list, keytag| The RRSIG (signature) with tag {keytag} for the NSEC3 record cannot be verified. Fetched from the nameservers with IP addresses "{ns_ip_list}".
 DS10_NSEC_ERR_TYPE_LIST            | ERROR   | ns_ip_list | NSEC record for the zone apex with incorrect type list. Fetched from nameservers with IP addresses "{ns_ip_list}".
-DS10_NSEC_MISMATCH_APEX            | ERROR   | ns_ip_list | NSEC record with a non-apex owner name, which is unexpected. Fetched from nameservers with IP addresses "{ns_ip_list}".
+DS10_NSEC_MISMATCH_APEX            | ERROR   | ns_ip_list | The returned NSEC record has an unexpected non-apex owner name. Fetched from nameservers with IP addresses "{ns_ip_list}".
```

Comment:
Because of the way NSEC and NSEC3, respectively, works. The NSEC record for apex has apex as owner name. The NSEC3 record for apex has a hash version of the apex name as owner name.

Reply:
The form with verb inflected in singular is directly matching the set. The form with the uninflected verb is a simplified form. I will make both inflected.